Tools/scripts/build_ci.sh

```diff
@@ -193,7 +193,7 @@ for t in $CI_BUILD_TARGET; do
 
     if [ "$t" == "configure-all" ]; then
         echo "Checking configure of all boards"
-        ./Tools/scripts/configure_all.py
+        ./Tools/scripts/configure_all.py --python=python3 --python2=python2
```

Comment:
Perhaps `--python-a` and `--python-b` would be clearer.

Reply:
we might want to use a version of python binary that actually isn't named python3 or python2.

Reply:
@bugobliterator this is simply an invocation of the script which is passing through python3 as primary and python2 as secondary.  That's entirely appropriate for CI, which is what this script is all about :+1: